packages/teams-js/src/internal/utils.ts

```diff
@@ -54,18 +62,28 @@ export function validateOrigin(messageOrigin: URL): boolean {
     return false;
   }
   const messageOriginHost = messageOrigin.host;
-
-  if (validOrigins.some((pattern) => validateHostAgainstPattern(pattern, messageOriginHost))) {
-    return true;
+  let isValid = Communication.validatedDomains.get(messageOrigin.host);
```

Comment:
>let isValid = Communication.validatedDomains.get(messageOrigin.host); [](http://example.com/codeflow?start=2&length=69)

It seems like this code would be clearer if `isValid` was always a `boolean` value and not sometimes undefined (despite the fact that JS is fast and loose with the relationship between `undefined` and `false`).

The usage of `isValid` would probably be clearer if this line was written like this:
```
let isValid = Communication.validatedDomains.get(messageOrigin.host) ?? false;
```